admin-app/src/main/kotlin/com/epam/drill/admin/DrillAdminApplication.kt

```diff
@@ -46,6 +52,8 @@ import javax.sql.DataSource
 private val logger = KotlinLogging.logger {}
 
 fun Application.module() {
+    val oauth2Enabled = oauth2Enabled
```

Comment:
reason for creating vals

Reply:
gotta use that since we don't have `Application` context inside module definitions below (with `di`,etc)